nestjs-app/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Test coverage files, generated when you run `yarn test:cov`
```

Comment:
I added each of "coverage", "dist", and "node_modules" manually, because it seemed like these are the types of files we want to keep out of the repo (generated files / dependency binaries). But I'd love if someone could sanity check my thinking here!